canonical/canonical.com
# feat: /openstack landing page

**Note: form will be added in separate PR**

## Done

- Built page as per [doc](https://docs.google.com/document/d/1v1G2y465b_VYVUG9TIV-A_X1dU4kg5Zvrwd2latidCk/edit?tab=t.0) and [design](https://www.figma.com/design/vH19y2J2YqwUfksebSUvTW/OpenStack?node-id=224-7535&node-type=instance&m=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1394.demos.haus/openstack
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14071
